src/views/Login.jsx

```diff
@@ -1,38 +1,73 @@
-import React from 'react'
-import { Avatar, Button, Grid, Paper, TextField, Typography, Link} from '@mui/material'
+import React, {useContext, useState} from 'react'
+import { Link } from 'react-router-dom';
+import { Avatar, Button, Grid, Paper, TextField, Typography } from '@mui/material'
 import LockOutlinedIcon from '@mui/icons-material/LockOutlined';
 import { red } from '@mui/material/colors';
+import { AuthContext } from '../store/AuthContext';
+import Header from '../Components/Header';
 
 
-const Login = ({handleChange}) => {
-  const paperStyle={padding: 20, width: 300, height: "52vh", margin:"0 auto"}
-  const avatarStyle={backgroundColor:"#ff0000"}
-  const btnStyle={margin: "8px 0", backgroundColor: red[500]}
-  const linkStyle={margin: "8px"}
+const Login = () => {
+  const paperStyle={padding: 20, width: 300, height: "52vh", margin:"40px auto"};
+  const avatarStyle={backgroundColor:"#ff0000"};
+  const btnStyle={margin: "8px 0", backgroundColor: red[500]};
+  const linkStyle={margin: "8px"};
+
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+  const { login }=useContext(AuthContext);
+
+    const handleEmailChange = (e) => {
+        setEmail(e.target.value)
+    };
+
+    const handlePasswordChange = (e) => {
+        setPassword(e.target.value)
+    };
+
+    const handleRegister = (event) => {
+        // DO YOUR DUE DILIGENCE HERE: check email format, and maybe password length
+        event.preventDefault()
+        login(email, password)
+    };
   return(
+    <>
      <Grid>
+        < Header />
       <Paper style={paperStyle}>
           <Grid align="center">
           <Avatar style={avatarStyle}>
               <LockOutlinedIcon/>
            </Avatar>
               <h2>Login</h2>
           </Grid>
-          <TextField variant="standard" label="Username" placeholder="enter username"
-          fullWidth required/>
-          <TextField variant="standard" label="Password" placeholder="enter password"
+          <TextField 
```

Comment:
You could add an `onBlur` event and handle the logic to validate the email there. 
This would make the experience of the user better.

Reply:
But maybe that's out of the scope of this task.